### PR TITLE
O11Y-5307: Run CI Once

### PR DESCRIPTION
**Pull request by kennytrytek-wf**

### CODE REVIEW

File: .github/workflows/dart_ci.yaml [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 on:
+  workflow_dispatch:
   push:
     branches:
-      - 'master'
-      - 'test_consume_*'
+      - "master"
+      - "_integration/**/*"
+      - "experiments/**/*"
+      - "test_consume_**"
```
Comment (blakeroberts-wk) [inline .github/workflows/dart_ci.yaml:9]: Are any of these used at all or should we just do master?

Comment (kennytrytek-wf) [reply .github/workflows/dart_ci.yaml:9]: No, they are not used, but it doesn't hurt anything to keep them available for use if somebody wants to use them. We already set it up this way for the other 34 repos.

Comment (kennytrytek-wf) [reply .github/workflows/dart_ci.yaml:9]: That's fine, opentelemetry-dart has its own use-cases that differ from other O11y repos.

**Review by blakeroberts-wk — APPROVED** [review]

**Comment by kennytrytek-wf** [comment]
QA +1 ⛄ 

@Workiva/release-management-pp

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**